install_sct

```diff
@@ -153,17 +153,25 @@ Please check your internet connection before relaunching the installer\n"
   fi
 }
 
-# Usage of this script
-function usage() {
-  echo -e "\nUsage: $0 [-d] [-b] [-v]" 1>&2
-  echo -e "\nOPTION"
-  echo -e "\t-d \v Prevent the (re)-installation of the \"data/\" directory "
-  echo -e "\n\t-b \v Prevent the (re)-installation of the SCT binaries files "
-  echo -e "\n\t-v \v Full verbose"
-}
-
 # SCRIPT STARTS HERE
 # ========================================================================
+ARGPARSE_ALLOW_UNKNOWN_ARGS=true # tell argparse to allow unknown args
+source scripts/argparse.bash
+argparse "$@" <<EOF || exit 1
+parser.add_argument('-d', '--no_data_install', action='store_true', help='-d \v Prevent the (re)-installation of the \"data/\" directory')
```

Comment:
i would suggest to replace underscore with hyphen (easier to type): `--no_data_install` --> `--no-data-install`

to apply for all flags

Reply:
Cool. 

Done!